src/main/java/com/github/stephengold/joltjni/IndexedTriangle.java

```diff
@@ -21,12 +21,15 @@ of this software and associated documentation files (the "Software"), to deal
  */
 package com.github.stephengold.joltjni;
 
+import com.github.stephengold.joltjni.readonly.ConstIndexedTriangle;
+
 /**
  * A triangle composed of three 32-bit mesh-vertex indices and a material index.
  *
  * @author Stephen Gold sgold@sonic.net
  */
-final public class IndexedTriangle extends IndexedTriangleNoMaterial {
+final public class IndexedTriangle extends IndexedTriangleNoMaterial
+        implements ConstIndexedTriangle {
     // *************************************************************************
     // constructors
```

Comment:
Now that this interface exists, `getMaterialIndex()` should be moved to a new section titled "ConstIndexedTriangle methods.